ledger/src/blockstore.rs

```diff
@@ -8284,6 +8235,35 @@ pub mod tests {
         );
     }
 
+    impl Blockstore {
+        pub(crate) fn write_deprecated_transaction_status(
```

Comment:
Huh, didn't know you could do this. Can `blockstore.write_deprecated_transaction_status()` be called outside of `#[cfg(test)]` ? I saw you're calling it from `blockstore_purge.rs` as well.

Reply:
It is not callable outside `#[cfg(test)]`, which means it wouldn't be usable in a crate-level integration test nor in a unit test in a different crate. But it can be used across module unit tests :)